qtfred/src/ui/dialogs/ShipEditor/ShipEditorDialog.cpp

```diff
@@ -626,9 +627,33 @@ void ShipEditorDialog::on_weaponsButton_clicked()
 }
 void ShipEditorDialog::on_playerOrdersButton_clicked()
 {
-	auto dialog = new dialogs::PlayerOrdersDialog(this, _viewport, getIfMultipleShips());
-	dialog->setAttribute(Qt::WA_DeleteOnClose);
-	dialog->show();
+	// auto dialog = new dialogs::PlayerOrdersDialog(this, _viewport, getIfMultipleShips());
+	// dialog->setAttribute(Qt::WA_DeleteOnClose);
+	// dialog->show();
```

Comment:
Placeholder stuff for the follow-up weapons revision?

Reply:
Nah leftovers from where the removed dialogs were. These can be removed safely